File: java/07 - Demo 3 - Refactoring/src/test/java/renamemethod/TelephoneNumberTest.java
package renamemethod;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class TelephoneNumberTest {

	private static final String OFFICE_NUMBER = "154689";
	private static final String OFFICE_AREA_CODE = "016";

	@Test
	public void testGetTelephoneNumber() {
		Person person = new Person(OFFICE_AREA_CODE, OFFICE_NUMBER);
		
		String result = person.getNumber();
		assertEquals("(" + OFFICE_AREA_CODE + ") " + OFFICE_NUMBER, result);
	}
}

File: csharp/07 - Demo 3 - Refactoring/Refactoring.Tests/RenameMethod/TelephoneNumberTest.cs
using Xunit;
using FluentAssertions;
using Refactoring.RenameMethod;

namespace Refactoring.Tests.RenameMethod
{
    public class TelephoneNumberTest
    {
        private const string OFFICE_NUMBER = "154689";
        private const string OFFICE_AREA_CODE = "016";

        [Fact]
        public void GetTelephoneNumber()
        {
            Person person = new Person(OFFICE_AREA_CODE, OFFICE_NUMBER);

            var result = person.TelephoneNumber;
            result.Should().Be("(" + OFFICE_AREA_CODE + ") " + OFFICE_NUMBER);
        }
    }
}